lib/drivers/node-mongodb-native/connection.js

```diff
@@ -41,6 +41,9 @@ NativeConnection.prototype.__proto__ = MongooseConnection.prototype;
  */
 
 NativeConnection.prototype.useDb = function(name) {
+  // Return immediately if existed
+  if (this.relatedDbs[name]) return this.relatedDbs[name];
```

Comment:
Would you be open to putting this behind a flag? I'm wary because this could very likely break peoples code

Reply:
@rocketspacer any updates?